docs/JestObjectAPI.md

```diff
@@ -0,0 +1,204 @@
+---
+id: jest-object
+title: The Jest Object
+layout: docs
+category: API Reference
+permalink: docs/jest-object.html
+next: configuration
+---
+
+The methods in the global `jest` object help create mocks and let you control Jest's overall behavior.
```

Comment:
`jest` is not a global, see @cpojer's [comment](https://github.com/facebook/jest/pull/2388#discussion_r93334960).